pkg/vm/slot.go

```diff
@@ -29,30 +29,30 @@ func (s slot) Set(i int, item stackitem.Item, refs *refCounter) {
 }
 
 // Get returns the item contained in the i-th slot.
-func (s slot) Get(i int) stackitem.Item {
+func (s Slot) Get(i int) stackitem.Item {
 	if item := s[i]; item != nil {
 		return item
 	}
 	return stackitem.Null{}
 }
 
-// ClearRefs removes all slot variables from the reference counter.
-func (s slot) ClearRefs(refs *refCounter) {
+// clearRefs removes all slot variables from the reference counter.
+func (s Slot) clearRefs(refs *refCounter) {
 	for _, item := range s {
 		refs.Remove(item)
 	}
 }
 
 // Size returns the slot size.
-func (s slot) Size() int {
+func (s Slot) Size() int {
 	if s == nil {
-		panic("not initialized")
+		return 0
```

Comment:
I'm OK with this change. @roman-khimov do you think it's OK to return 0 for uninitialized slot?

I thought about returning an error, but it will complicate method's signature for nothing. And we don't use this method inside VM anyway, hence it does not affect the node state in any way.